osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -388,60 +412,32 @@ - (BOOL)string:(NSString *)string containsAllCharactersInString:(NSString *)char
 }
 
 
-//<##>runRootCommands
-BOOL runRootCommands(NSString *command) {
-//    NSString *msg = [NSString stringWithFormat:@"Running root command/s: %@...", command];
-//    showNotification(msg);
-    
-    NSDictionary *errorInfo = runAsRoot(command);
-    if (errorInfo != nil) {
-//        msg = [NSString stringWithFormat:@"FAILED command/s %@ with ERROR: %@", command, errorInfo];
-//        showNotification(msg);
-        return FALSE;
-    }
-//    msg = [NSString stringWithFormat:@"Done running root command/s %@.", command];
-//    showNotification(msg);
-    return TRUE;
-}
-
-
-//<##>setLaunchAgent
-NSString *getLaunchAgentCommand(NSString *source, NSString *target) {
-    if (pathExists(source)) {
-        return [NSString stringWithFormat:@"cp '%@' '%@'", source, target];
-    }
-    return nil;
+NSString *getUsrBinKalite() {
+    return @"/usr/local/bin/kalite";
```

Comment:
Sorry, let's use `/usr/bin/kalite` for now to be consistent with previous releases.  Let's address using `/usr/local/bin/kalite` in another issue.